modules/@apostrophecms/area/ui/apos/components/AposAreaExpandedMenu.vue

```diff
@@ -6,7 +6,6 @@
     @inactive="modal.active = false"
     @show-modal="modal.showModal = true"
     @esc="close"
-    @no-modal="$emit('safe-close')"
```

Comment:
This has been removed because people using modals forget to call it.
The safe-close event is emitting in `AposModal` when it closes.
I kept the event `no-modal` to avoid BC.